javascript_client/subscriptions/ActionCableLink.js

```diff
@@ -0,0 +1,41 @@
+var ApolloLink = require("apollo-link").ApolloLink
+var Observable = require("apollo-link").Observable
+var printer = require("graphql/language/printer")
+
+function ActionCableLink(options) {
+  var cable = options.cable
+  var channelName = options.channelName || "GraphqlChannel"
+  var actionName = options.actionName || "execute"
+
+  return new ApolloLink(function(operation) {
+    return new Observable(function(observer) {
+      var subscription = cable.subscriptions.create(channelName, {
```

Comment:
I think we'll need a "channel id" here, something to distinguish this subscription from previous ones. Otherwise, we can't have two open subscriptions at a time. 

I learned this the hard way with the previous Apollo subscriber: 

https://github.com/rmosolgo/graphql-ruby/blob/29e28627dfc4254a6fc68ebf602b56cc354e9eb9/javascript_client/subscriptions/ActionCableSubscriber.js#L24-L30

Sorry there's not a proper way to test this, but it's demonstrated in the system tests: 

https://github.com/rmosolgo/graphql-ruby/blob/29e28627dfc4254a6fc68ebf602b56cc354e9eb9/spec/dummy/app/assets/javascripts/application.js#L35-L41

Reply:
copy pasted